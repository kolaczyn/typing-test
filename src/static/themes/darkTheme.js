import { DARK_THEME } from '../../reducers/theme/actions';

const darkTheme = {
  palette: {
    box: '#222',
    border: '#1c1e21',
    background: '#303030',
    text: '#fff',

    green: '#55efc4',
    blue: '#00a1e4',
    red: '#e17055',
    yellow: '#e8e9a1',
    lightGray: 'bisque',
  },
  themeName: DARK_THEME,
};

export default darkTheme;
