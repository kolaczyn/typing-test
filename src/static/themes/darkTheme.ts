import Action from '../../reducers/theme/actions';
import ThemeType from './ThemeType';

// https://flatuicolors.com/palette/defo
const darkTheme: ThemeType = {
  palette: {
    box: '#2c3e50',
    border: '#27394B',
    background: '#34495e',
    text: '#ecf0f1',

    green: '#2ecc71',
    blue: '#3498db',
    red: '#f96454',
    yellow: '#f1c40f',
    lightGray: '#bdc3c7',
    darkGray: '#bdc3c7',
  },
  themeName: Action.DARK_THEME,
};

export default darkTheme;
