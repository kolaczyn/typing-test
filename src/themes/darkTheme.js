export default {
  palette: {
    box: '#222',
    background: '#303030',
    lightGray: 'bisque',
    green: '#55efc4',
    red: '#e17055',
    text: '#fff',
  },
  containerWidth: '768px',
};
