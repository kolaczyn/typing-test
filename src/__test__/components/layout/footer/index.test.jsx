import React from 'react';
import { ThemeProvider } from 'styled-components';
import { cleanup, render } from '@testing-library/react';

import Footer from '../../../../components/layout/footer';
import darkTheme from '../../../../static/themes/darkTheme';

afterEach(cleanup);

describe('<Navbar />', () => {
  it('renders <Navbar />', () => {
    const { container } = render(
      <ThemeProvider theme={darkTheme}>
        <Footer />
      </ThemeProvider>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
