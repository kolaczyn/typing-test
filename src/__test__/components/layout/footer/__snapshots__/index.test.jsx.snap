// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> renders <Navbar /> 1`] = `
<footer
  class="styles__Wrapper-yt96dg-0 iYiSqY"
>
  <div
    class="container-tfwxup-0 PjAJF"
    style="margin: 0px auto;"
  >
    <section
      class="styles__Copyright-yt96dg-2 jraqFv"
    >
      <span>
        © 2021
      </span>
       
      <span>
        created by Paweł Kołaczyński
      </span>
    </section>
    <ul
      class="styles__LinksWrapper-yt96dg-1 WAOqC"
    >
      <li>
        <a
          class="styles-ja41pm-0 elKeK"
          href="https://github.com/kolaczyn/typing-test"
          label="github"
          target="_blank"
          to=""
        >
          <i
            class="fab fa-github"
          />
          <p>
            github
          </p>
        </a>
      </li>
      <li>
        <a
          class="styles-ja41pm-0 elKeK"
          href="https://twitter.com/kolaczyn"
          label="twitter"
          target="_blank"
          to=""
        >
          <i
            class="fab fa-twitter"
          />
          <p>
            twitter
          </p>
        </a>
      </li>
      <li>
        <a
          class="styles-ja41pm-0 elKeK"
          href="mailto:kolaczyn@protonmail.com"
          label="email"
          to=""
        >
          <i
            class="fas fa-envelope"
          />
          <p>
            email
          </p>
        </a>
      </li>
      <li>
        <a
          class="styles-ja41pm-0 elKeK"
          href="https://kolaczyn.com"
          label="website"
          target="_blank"
          to=""
        >
          <i
            class="fas fa-code"
          />
          <p>
            website
          </p>
        </a>
      </li>
    </ul>
  </div>
</footer>
`;
